src/assets/i18n/pt.json

```diff
@@ -19,5 +19,13 @@
     "id": "Id",
     "name": "Nome",
     "filter": "Filtro"
+  },
+  "form": {
+    "add": "Adicione um computador",
+    "update": "Atualize um computador"
+  },
+  "formCompany" : {
```

Comment:
FormCompany in from and add another level : form.computer.add and form.company.add